server/src/test/java/umm3601/user/UserControllerSpec.java

```diff
@@ -133,20 +138,29 @@ public void canGetUsersWithGivenAgeAndCompany() throws IOException {
       assertEquals(25, user.age);
       assertEquals("OHMNET", user.company);
     }
+    assertEquals(1, argument.getValue().length);
   }
 
   @Test
   public void canGetUserWithSpecifiedId() throws IOException {
-    when(ctx.pathParam("id")).thenReturn("588935f5c668650dc77df581");
+    String id = "588935f5c668650dc77df581";
+    User user = db.getUser(id);
+
+    when(ctx.pathParam("id")).thenReturn(id);
+
     userController.getUser(ctx);
-    verify(ctx).status(HttpCode.OK);
+
+    verify(ctx).json(user);
+    verify(ctx).status(HttpStatus.OK);
+    assertEquals("Cervantes Morin", user.name);
```

Comment:
```suggestion
```

Reply:
Do you think it's at all important that we know that the id used earlier in the test matches up with one of the users? Or, is that obvious since lines 153 and 154 succeed?